girder-dandi-archive/girder_dandi_archive/rest.py

```diff
@@ -203,15 +198,14 @@ def list_dandisets(self, limit, offset, sort):
     )
     def search_dandisets(self, search, limit, offset, sort):
         # Ensure we are only looking for drafts collection child folders.
-        drafts = get_or_create_drafts_collection()
         # TODO Currently only searching identifier, name, description, and contributor name
```

Comment:
what is the TODO here exactly?

Reply:
That's me, basically it's `search better`. I wanted to leave a note describing the current behavior so whoever gets to do the full search function knows what's up.

Reply:
Could we remove this comment in that case? There isn't something actionable here and this suggests to me when reading the code that something is to be done.